app/main.py

```diff
@@ -1,3 +1,82 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+import datetime
+import os
+from typing import List
+from app.customer import Customer
+from app.shop import Shop
+from app.car import Car
+
+
+def get_food_cost(
+        bayer: Customer,
+        shops: List[Shop],
+        fuel_price: float
+) -> Shop | float | float:
```

Comment:
It return tuple not one
```suggestion
) -> tuple[Shop, float, float]:
```